plugin/user_plugin_server.py

```
@@ -0,0 +1,576 @@
+                    key1 = f"{pid}.{eid}"
+                    key2 = f"{pid}:plugin_entry:{eid}"
+                    # 构造 EventHandler 并注册（最后注册的覆盖同名）
+                    _event_handlers[key1] = EventHandler(meta=event_meta, handler=member)
+                    _event_handlers[key2] = EventHandler(meta=event_meta, handler=member)
+                    # 记录 (plugin_id, entry_id) -> python method 名，供触发时服务器端回退使用
+                    try:
+                        _plugin_entry_method_map[(pid, str(eid))] = name
+                    except Exception:
+                        logger.debug("Failed to map entry method for %s.%s", pid, eid, exc_info=True)
+
+                # 新增：基于 plugin.toml 中列出的 entries（如果有），尝试为实例中对应的方法自动注册 EventHandler
```

Ruff (0.14.7)

Comment contains ambiguous `：` (FULLWIDTH COLON). Did you mean `:` (COLON)?

(RUF003)